buzzbooster-ios/BSTSample/AppDelegate.swift

```diff
@@ -0,0 +1,60 @@
+import UIKit
+import BuzzBooster
+import UserNotifications
+import Toast
+
+@main
+class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterDelegate {
+  static var APP_KEY: String = "0d944ac8a05b"
+  static var USER_ID: String = NSUUID().uuidString
+
+  var window: UIWindow?
+  
+  func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+    UNUserNotificationCenter.current().delegate = self;
```

Comment:
nit. 이 부분 외에도 불필요한 semicolon은 삭제하면 좋을 것 같습니다!
```suggestion
    UNUserNotificationCenter.current().delegate = self
```